Automattic/wp-calypso
# Hosting import flow: reuse existing step and styles

Related to https://github.com/Automattic/wp-calypso/pull/77588

## Proposed Changes

* Address some review comments in https://github.com/Automattic/wp-calypso/pull/77588:
   - [Don't create a new import step ](https://github.com/Automattic/wp-calypso/pull/77588#discussion_r1219462250)
   - [Reuse CSS instead of adding new](https://github.com/Automattic/wp-calypso/pull/77588#discussion_r1217708702). This PR adds`.import-hosted-site` to `global.css`, which positions the progress bar correctly and doesn't require additional padding or adjustments. 

## Testing Instructions



* Try out the `/setup/import-hosted-site` 
  - Make sure you can go end-to-end and that you can still only use a URL and not the `ImportList` step
  - Check the styling at all breakpoints to make sure there is enough padding and that content wraps

## Pre-merge Checklist
